docs/standards/nft-2.0/LSP8-Identifiable-Digital-Asset.md

```diff
@@ -13,83 +13,105 @@ sidebar_position: 4
 
-### NFT Metadata
+![LSP8 Game Nested NFTs TokenIds Representation](/img/standards/lsp8-game.jpeg)
+### Unlimited Metadata
 
-One of the critical differences of LSP8 is that it allows representing each NFT as more than just a unique `tokenId` number. Each newly created `tokenId` (NFT) has its own metadata, that describes its uniqueness. Such metadata can refer to either:
+:::tip Recommendation
 
-- _off-chain metadata_ (a **JSON file URL** stored on IPFS, a public registry, or even permissioned or private servers) or
-- _on-chain metadata_ (within another **ERC725Y smart contract**).
+To mark the **asset authenticity**, it's advised to use a combination between **[LSP4-DigitalAssetMetadata](./LSP4-Digital-Asset-Metadata.md)** and **[LSP12-IssuedAssets](../universal-profile/lsp12-issued-assets.md)**.
 
-It is possible to know where the metadata for a specific `tokenId` is located by simply querying the LSP8 contract, giving the `tokenId` as an argument.
+:::
 
-One benefit of using an ERC725Y metadata contract per singular `tokenId` or NFT is that users can edit the metadata related to this NFT. Changeability makes the metadata flexible and upgradable. It enables the representation of unique NFTs that can be altered over time.
+The current token standards don't enable attaching metadata to the contract in a generic and flexible way, they set the **name**, **symbol**, and **tokenURI**. This is limiting for a digital asset that may want to list the creators, the community behind it, and to have the ability to update the metadata of the token and the tokenIds over time depending on a certain logic (Evolving tokens).  
 
-## Types of `tokenId`
+To ensure a flexible and generic asset representation, the token contract should use the **[LSP4-DigitalAsset-Metadata](./LSP4-Digital-Asset-Metadata.md)**. In this way, any information could be attached to the token contract.
 
-:::caution
+:::note Notice
 
-It is recommended that the `tokenId` should not change over the lifecycle of the LSP8 contract.
+The Metadata defined by the **ERC725Y Data Keys** can be set for **each tokenId**, not just for the whole NFT collection. Check **[LSP8 ERC725Y Data Keys](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#erc725y-data-keys)** in the LSP8 Standard Document.
 
 :::
 
-An LSP8 NFT can be represented in multiple ways, depending on the type of `tokenId` used as:
+### Force Boolean
 
-- a **number** (`uint256`) that increments on each newly minted NFT,
-- a unique `bytes32` value (representing a _serial number_, for instance), or
-- an ERC725Y contract address.
+It is expected in the LUKSO's ecosystem to use **smart contract based accounts** to operate on the blockchain, which includes receiving and sending tokens. EOAs can receive tokens but they will be mainly used to control these accounts and not to hold tokens.
```

Comment:
Maybe we can add a link to LSP0 docs page for **smart contract-based accounts**?

```suggestion
It is expected in the LUKSO's ecosystem to use **smart contract-based accounts** to operate on the blockchain, which includes receiving and sending tokens. EOAs can receive tokens, but they will be used mainly to control these accounts and not to hold them.
```